README.md

```diff
@@ -43,6 +43,47 @@ To uninstall again, type
 $ pip uninstall pybamm
 ```
 
+## Optional dependencies
+
+### [scikits.odes](https://github.com/bmcage/odes)
+
+Users can install [scikits.odes](https://github.com/bmcage/odes) in order to use the
+wrapped SUNDIALS ODE and DAE
+[solvers](https://pybamm.readthedocs.io/en/latest/source/solvers/scikits_solvers.html).
+To install scikits.odes, either follow the instructions below, or use the provided
+install script in `scripts/install_scikits_odes.sh`.
```

Comment:
Not sure how detailed these instructions should be (or if these are intended as the instructions for others in Faraday) but I think it would be easier for others in Faraday if there was exact commands to enter e.g 
``` 
cd PyBaMM
source scripts/install_scikits_odes.sh
```
and then have the instruction to put the sundials path in venv/bin/activate or .bashrc here aswell. 

It wasn't obvious to me reading this whether adding the sundials path is part of the alternative installation or part of both.

Reply:
Ok, I'll make it clear that users should always add the sundials path LD_LIBRARY_PATH, whether they run the script or not.

I didn't really want users blindly running that script, as it does a pip install, the result of which is going to be rather different if they are in/out of a virtualenv. They might even prefer conda, depending on their setup. At least if they are copying one line at a time, they will see the pip install and adjust accordingly

Reply:
how is this @Scottmar93?  I removed the pointer to `scrips/install_scikits_odes.sh` as well, thinking about it more I think it would be a bad idea to get users to run this

Reply:
I think this looks great and easy to follow. I hadn't really thought about different setups etc so seems sensible to remove a pip specific script.